contract_compassion/tests/test_base_module.py

```diff
@@ -20,8 +20,8 @@ class test_base_module(test_base_contract):
 
     def setUp(self):
         # Retrieve of income account
-        account_obj = self.env['account.account']
-        property_account_income = account_obj.search([
+        super(test_base_module, self).setUp()

```

Comment:
oh my bad i didn't understood the question well, i thought the question was why i haven't inherit from test_recurring_contract